### ISSUE TITLE
Allow increasing volume over 100%

### ISSUE THREAD

**Issue by BrownNPC** [recloudstream/cloudstream#1086]
### Describe your suggested feature

A few videos have low volume. A toggle could be added to the player settings that removed the 100% limit. It would make the user experience better.

### Other details

Vlc cannot be used for most videos , because the https stream gets blocked on it. What i'm trying to say is "Just use vlc" is not a valid option.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Blatzar** [comment]
> What i'm trying to say is "Just use vlc" is not a valid option. 

:neutral_face: :neutral_face: :neutral_face: :neutral_face: :neutral_face: :neutral_face: :neutral_face: :neutral_face: 

https://github.com/google/ExoPlayer/issues/5282#issuecomment-449655159 was the best I could find and it does not seem unnecessarily complex.

**Comment by Luna712** [comment]
I may make an attempt at this. I did have something that partially worked awhile ago actually but I hated my implementation method and would rather bot have it then how I did it so kinda abandoned it, but I may make another better attempt at this again.

**Comment by Luna712** [comment]
Just an FYI, I finally completed a PR (#1444) for this after numerous try-fail attempts that both works and (in my opinion) has pretty good UI for displaying boost volume as well. So once that is merged this will be done.
